### PR TITLE
Add mi2 2.1.12r5

### PR DESCRIPTION
**Pull request by kkdd**
This formula is an old caskroom/mi.rb (version 2).

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/mi2.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'mi' do
+  version '2.1.12r5'
+  sha256 'f594e33451fd70cf16c36af205f59294609d910836a91c877e9d2f74261b5d38'
```
Comment (commitay) [inline Casks/mi2.rb]: This is the same as the current `mi` Cask
https://github.com/Homebrew/homebrew-cask/blob/master/Casks/mi.rb
```
cask 'mi' do
  version '2.1.12r5'
  sha256 'f594e33451fd70cf16c36af205f59294609d910836a91c877e9d2f74261b5d38'
```

Comment (kkdd) [reply Casks/mi2.rb]: The current caskroom/mi.rb (version 2) is old, which I will edit toward version 3. Please wait for minutes.

**Comment by commitay** [comment]
@kkdd Is there a specific reason you need this Cask?

**Comment by commitay** [comment]
No response, closing.

**Pull request closed without merging**